readme.md

```diff
@@ -24,6 +24,13 @@ import {patternToRegex} from 'webext-patterns';
 
 ## Usage
 
+The package offers 2 exports:
+
+- `patternToRegex`, for [patterns](https://developer.chrome.com/extensions/match_patterns), which are used by manifest permissions and content scripts’ `matches`
+- `globToRegex`, for [globs](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/content_scripts#globs), which are used by additional, userscripts-compatible filters like the content scripts’ `include_globs` and `exclude_globs`.
```

Comment:
I replace the URL here because browsers do not support "regex as globs" as documented in the previous URL: https://wiki.greasespot.net/Include_and_exclude_rules